org.ekstep.lessonbrowser-1.4/editor/lessonBrowserApp.js

```diff
@@ -422,7 +422,8 @@ angular.module('org.ekstep.lessonbrowserapp', ['angular-inview', 'luegg.directiv
                         var contents = [];
                         $scope.mainTemplate = 'facetsItemView';
                         ctrl.applyAllJquery();
-                        if (ctrl.facetsResponse.result.response.sections) {
+                        var contentCount = _.findIndex(ctrl.facetsResponse.result.response.sections, function(o) { return o.count > 0 });
+                        if (ctrl.facetsResponse.result.response.sections && contentCount > 0) {
```

Comment:
Going forward please use `_.get()` Instead of having multiple checks

Reply:
Is this _.get() works for array??

Reply:
Only works for objects

Reply:
ctrl.facetsResponse.result.response.sections is array, so this will not work.